frontend/src/app/user/master-screener/questions/questions.component.css

```diff
@@ -3,7 +3,7 @@ main {
     flex-direction: column;
     justify-content: space-between;
     overflow-x: hidden;
-    padding-bottom: 2em;
+    padding-bottom: 20em;
```

Comment:
That's a lot of padding!  You are sure this is right?

Reply:
Yes. I think this was a typo as when the data is initialized, the padding is set to 20.